cryptonight/src/cnaes.rs

```diff
@@ -302,8 +300,8 @@ const CRYPTONIGHT_SBOX: [u8; 4096] = [
 ];
 
 const fn substitute_word(word: u32) -> u32 {
-    let wb: [u8; 4] = word.to_be_bytes();
-    u32::from_be_bytes([
+    let wb: [u8; 4] = word.to_le_bytes();
+    u32::from_le_bytes([
```

Comment:
Since big endian computers are almost non-existent these days, I reworked the AES functions to use LE operations everywhere when going between bytes and words.